log4j-core/src/test/java/org/apache/logging/log4j/core/LookupTest.java

```diff
@@ -16,14 +16,15 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
static imports go last. We allow wildcard static imports so this change isn't necessary.